src/invalidations.jl

```diff
@@ -255,6 +255,41 @@ else
     new_backedge_table() = Dict{Tuple{Int32,UInt64},Tuple{Any,Vector{Any}}}()
 end
 
+"""
+    report_invalidations(;
+        job_name::String = "",
+        invalidations,
+        n_rows::Int = 10,
+        process_filename::Function = x -> x,
+    )
+
+Print a tabular summary of invalidations given:
+
+ - `job_name::String` the name of the job
```

Comment:
What's the role of the job_name? Does it have any functional significance? (Can I choose a name that's wrong or broken?)

Reply:
You know, I think we can get rid of it. It's not really doing anything. I added it in case I'd run several invalidation jobs, so that the tables could be differentiated, but 1) that can be printed before the call to `report_invalidations` and 2) I think they'd need to be separate sessions anyway. I'll remove it